recipes/configobj-dev/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "configobj-dev" %}
+{% set version = "5.1.0.dev20190901" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: 48a8d93afc971a16e43ba3d831a65403642430f806986d5038e5e1378aa3d7ee
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - configobj
+    - configobj.validate
+
+about:
+  home: https://github.com/hoechenberger/configobj-dev
```

Comment:
This is a fork? What is the motivation for having this?

Reply:
I have to admit the constellation is a little odd and I should have explained from the beginning.

So the latest release of the [`configobj`](https://github.com/DiffSK/configobj) package on PyPI is from 2014. Since then, numerous important fixes have been added to its development branch, but the maintainers have been unable to make  new release since (it's been promised for over 3 years now). Besides, the latest release on PyPI is broken and deviates from the release on GitHub with the same version number. See e.g. https://github.com/DiffSK/configobj/issues/160

This mess forces many users to use the GitHub `master` branch instead, which is less than optimal; other even started vendoring patched versions of the code.

I decided to bite the bullet and "fork" the project in the sense that it's a "packaging fork": the idea is to stay in sync with upstream, but to release packages to PyPI and `conda-forge` once in the while, at least until the upstream devs start publishing releases again (which may never happen).

Since the intention (at least so far) is not to extend the functionality of the project, but to simply package more of the recent, but unreleased upstream development efforts, I decided to name the fork `configobj-dev`.

I acknowledge the whole situation is a little unfortunate, but I trust this package could help many people

Reply:
Thanks for explaining the situation clearly, I had seen the recent commits to master then stopped looking further.

I think the current version number might cause issues. As you control the repository, what do you think about using a date based version number like `2019.09.01` instead?

Reply:
That should be possible, although I believe setuptools is going to strip the leading zeroes (so we would end up with 2019.9.1) but that shouldn’t cause any issues...

Reply:
Yes, you're right, I should have written `2019.9.1`.

Reply:
Just to clarify, `5.1.0-20190901` also wouldn't work? Because this one I could have `setup.py` generate automatically based on the current date :)

Reply:
(btw, `5.1.0.dev20190901` _is_ PEP-comliant :) But I don't know if/how well `conda` can handle version strings like this!)

Reply:
Thinking about it, a purely date-based version number might be the best choice anyway.  Will work on that and ping you when I'm done.

Reply:
[Dashes are forbidden](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html#info-index-json) as they're used to separate the name, version and build string so `.dev` would be better.

Does PyPI classify `5.1.0.dev20190901` as a development release that needs to be installed with `pip install --pre`? If not, I'm happy to merge this as is.

Reply:
@chrisburr I switched to a purely date-based versioning scheme now (`2019.9.1`). Once CI is finished, you can merge this :)